apps/products/models.py

```diff
@@ -33,3 +34,22 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class ProductImage(base_models.BaseModel):
+    product = models.ForeignKey(Products, on_delete=models.CASCADE)
+    product_image = models.ImageField(upload_to="profile")
+
+    class Meta:
+        ordering = ("created_at",)
+
+
+class Favorite(base_models.BaseModel):
+    product = models.ForeignKey(Products, on_delete=models.CASCADE)
```

Comment:
Add `related_name` the this product field to make it easy for us to access favorites on the product model